### PR TITLE
Adds prescription to loadout

### PR DESCRIPTION
**Pull request by Casper3667**
This adds a set of RP prescription medicines to the loadout. The full list is over at https://forums.aurorastation.org/topic/15809-feedback-loadout-prescriptions/ where a feedback thread has been made to adjust this appropriate.

- [x] Decide on dosage
- [x] Decide on pill amount
- [x] Finalise the medicine list

### CODE REVIEW

**Comment by Casper3667** [comment]
!wip

**Comment by Casper3667** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_utility.dm [hunk 1]
```diff
@@ -112,4 +112,19 @@
...
+	gear_tweaks += new /datum/gear_tweak/paper_data()
+
+/datum/gear/utility/pills
+	display_name = "pill bottle selection"
+	description = "A selection of prescription pill. NOTICE: most of these are considered contraband if possessed without the relevant prescription noted in the medical records."
+	path = /obj/item/reagent_containers/pill
+
+/datum/gear/utility/pills/New()
+	..()
+	var/pills = list()
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_utility.dm:124]:
```suggestion
	var/list/pills = list()
```

File: code/modules/client/preference_setup/loadout/loadout_utility.dm [hunk 2]
```diff
@@ -112,4 +112,19 @@
...
+	pills["Emoxanyl"] = /obj/item/storage/pill_bottle/emoxanyl
+	pills["Minaphobin"] = /obj/item/storage/pill_bottle/minaphobin/small
+	pills["Neurostabin "] = /obj/item/storage/pill_bottle/neurostabin
+	pills["Orastabin"] = /obj/item/storage/pill_bottle/orastabin
+	pills["Parvosil"] = /obj/item/storage/pill_bottle/parvosil
+	gear_tweaks += new/datum/gear_tweak/path(pills)
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_utility.dm:130]:
```suggestion
	gear_tweaks += new /datum/gear_tweak/path(pills)
```

**Review by Geevies — APPROVED** [review]

**Review by fernerr — APPROVED** [review]

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -134,16 +134,52 @@
...
-	name = "5u Minaphobin Pill"
+	name = "2u Minaphobin Pill"
 	desc = "Used to treat anxiety disorders and depression."
 	icon_state = "pill8"
-	reagents_to_add = list(/decl/reagent/mental/minaphobin = 5)
+	reagents_to_add = list(/decl/reagent/mental/minaphobin = 2)
+
+/obj/item/reagent_containers/pill/nerospectan
+	name = "2u Nerospectan Pill"
+	desc = "Used to treat a large variety of disorders including tourette, depression, anxiety and psychoses."
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/pill.dm:156]:
```suggestion
	desc = "Used to treat a large variety of disorders including tourettes, depression, anxiety and psychosis."
```

File: code/modules/reagents/reagent_containers/pill.dm [hunk 4]
```diff
@@ -134,16 +134,52 @@
...
+	icon_state = "pill8"
+	reagents_to_add = list(/decl/reagent/mental/nerospectan = 2)
+
+/obj/item/reagent_containers/pill/neurapan
+	name = "2u Neurapan Pill"
+	desc = "Used to treat a large variety of disorders including tourette, depression, anxiety and psychoses."
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/pill.dm:162]:
```suggestion
	desc = "Used to treat a large variety of disorders including tourettes, depression, anxiety and psychosis."
```

File: code/modules/reagents/reagent_containers/pill.dm [hunk 5]
```diff
@@ -134,16 +134,52 @@
...
+	icon_state = "pill8"
+	reagents_to_add = list(/decl/reagent/mental/neurapan = 2)
+
+/obj/item/reagent_containers/pill/neurostabin
+	name = "2u Neurostabin Pill"
+	desc = "Used to treat psychoses and muscle weakness."
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/pill.dm:168]:
```suggestion
	desc = "Used to treat psychosis and muscle weakness."
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Pull request merged**